### PR TITLE
Merge editorialized release notes for 20.0

### PR DESCRIPTION
**Pull request by mokagio**
Notice that no code change has landed in the release branch since the code freeze, so this PR doesn't include the usual changes we see when merging a new beta into `trunk`.

## Regression Notes

1. Potential unintended areas of impact – N.A.
2. What I did to test those areas of impact (or what existing automated tests I relied on) – N.A.
3. What automated tests I added (or what prevented me from doing so) – N.A.

---

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes. **N.A.**
- [x] I have considered adding accessibility improvements for my changes. **N.A.**
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary. **N.A.**

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -7,7 +7,6 @@
 * [*] Quick Start: The "Get to know the WordPress app" card has a fresh new look [#18688, #18747]
 * [*] Block Editor: A11y: Improve text read by screen readers for BottomSheetSelectControl [https://github.com/WordPress/gutenberg/pull/41036]
 * [*] Block Editor: Add 'Insert from URL' option to Image block [https://github.com/WordPress/gutenberg/pull/40334]
-* [*] Quick Start: The "Get to know the WordPress app" card has a fresh new look [#18688]
```
Comment (mokagio) [inline RELEASE-NOTES.txt:10]: I forgot to update this file when making https://github.com/wordpress-mobile/WordPress-iOS/commit/4ec2364eca97b11dd5ae237e3e71494f8c58833b 🤦‍♂️ Thankfully, @tiagomar and @momo-ozawa were onto it.

Comment (AliSoftware) [reply RELEASE-NOTES.txt:10]: 👍

File: WordPress/Jetpack/Resources/AppStoreStrings.po [hunk 2]
```diff
@@ -38,22 +38,15 @@ msgctxt "app_store_keywords"
...
 msgstr ""
 
-msgctxt "v19.9-whats-new"
+msgctxt "v20.0-whats-new"
 msgid ""
-"Blocks are now drag-and-drop, so you can easily rearrange your content however you like. We like to move it, move it.\n"
+"We’re kicking off version 20 with a fresh new look. The Quick Start card and Next Steps tours now match the new tour designs for existing sites, including a snazzy illustration. High five.\n"
 "\n"
-"That’s not the only change in the block editor. We adjusted color contrast so it’s easier to read text when you’re working in dark mode. We also fixed a problem with the Link To setting in Gallery blocks, and we added Image Size settings while we were at it.\n"
+"You can now insert images from other websites using the “Insert from URL” option in the app. More cat photos, please.\n"
```
Comment (AliSoftware) [inline WordPress/Jetpack/Resources/AppStoreStrings.po:45]: 🐈🐱😻😸

File: fastlane/Pluginfile [hunk 3]
```diff
@@ -7,8 +7,8 @@ group :screenshots, optional: true do
 end
 
 # This comment avoids typing to switch to a development version for testing.
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit', branch: 'trunk'
-# gem 'fastlane-plugin-wpmreleasetoolkit', '~> 4.1'
+# gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit', branch: 'trunk'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 4.2'
```
Comment (AliSoftware) [inline fastlane/Pluginfile:11]: Note: this change came from https://github.com/wordpress-mobile/WordPress-iOS/pull/18799 which I just approved and merged into the release branch

**Review by AliSoftware — APPROVED** [review]

**Pull request merged**